client/app/src/components/account/signing-tab.controller.js

```diff
@@ -61,7 +60,7 @@
             $mdDialog.hide()
           },
           function (error) {
-            accountCtrl.showMessage(error)
+            accountCtrl.showMessage(error)  // eslint-disable-line no-undef
```

Comment:
No reference to 'accountCtrl' here so this appears to be an issue that needs fixing as well